examples/dt/uni04delta-ipv6/README.md

```diff
@@ -0,0 +1,10 @@
+# Deployed Topology uni-delta
+
+**Based on OpenStack K8S operators from the "main" branch of the [OpenStack Operator repo](https://github.com/openstack-k8s-operators/openstack-operator/commit/7354503e770cbb0435700e2e5b2707de9f7d90e5)**
+
+## Purpose
+
+This is a twin of uni04delta, but it deploys IPv6 networks instead of IPv4.
+
+Please refer to that DT for details.
+
```

Comment:
If we add below config, neutron team can use uni04delta-ipv6 for ovn-ipv6 job.

#### OVN
OVNEncapTos=inherit
(needed for testcase test_dscp_inheritance_geneve)

Adding to testing tree
| Test framework   | When to run          | Special configuration |
| Tempest/neutron  | tempest stage       | Use rhel image        | 
| Tempest/scenario | tempest stage       | Use cirros image      |
| Rally (TBD)            | stage (TBD)       |                       |